config/initializers/postgresql_database_tasks.rb

```diff
@@ -0,0 +1,12 @@
+module ActiveRecord
+  module Tasks
+    class PostgreSQLDatabaseTasks
+      def drop
+        establish_master_connection
+        connection.select_all "select pg_terminate_backend(pg_stat_activity.pid) from pg_stat_activity where datname='#{configuration['database']}' AND state='idle';"
+        connection.drop_database configuration['database']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.